configure

```diff
@@ -4830,7 +4899,11 @@ EOF
 	cc_check $JPEG_CFLAGS $JPEG_LIBS -ljpeg && _jpeg=yes
 fi
 if test "$_jpeg" = yes ; then
-	append_var LIBS "$JPEG_LIBS -ljpeg"
+	if test "$_host_os" = kolibrios; then
```

Comment:
Instead of doing this check everywhere and as it's only a matter of adding a suffix, maybe it's easier to create a `_libext` variable set to `.dll` in the case of KOS and append it everytime we use the `-l` flag?
This would not work if the library is not for a DLL.

Another way would be to add `-ljpeg` to `JPEG_LIBS` when handling `--with-jpeg-prefix=`, setting it in KOS case with whatever is works with the toolchain.
Just before checking presence, add:
```
if [ -z "$JPEG_LIBS" ]; then
JPEG_LIBS=-ljpeg
fi
```
and remove -ljpeg in cc_check and in append_var.